sdk/tools/gen_baseaddress.py

```diff
@@ -379,7 +387,7 @@ def main():
     if len(dirs) < 1:
         trydir = os.getcwd()
         print(USAGE)
-        print('No path specified, trying the working directory: ', trydir)
+        print('No path specified, trying the working directory:', trydir)
```

Comment:
Is the space unnecessary?

Reply:
Yes: Python (comma) adds one automatically.

Reply:
ok I see...